src/signals/incident/components/form/DateTimeInput/styled.tsx

```diff
@@ -40,6 +40,6 @@ export const TimeWrapper = styled.div`
   }
 
   select {
-    width: 70px; /* fixed value from design */
+    width: 75px; /* fixed value from design */
```

Comment:
magic number ;-)

Reply:
That is why the comment is there; it explains where the value comes from.

Reply:
You don't need to change this, but to me the remark 'from design' is not really informative. I would assume that such a value is always included to conform with the design.